client/test/unit/specs/components/InterestModal.spec.js

```diff
@@ -240,6 +240,17 @@ describe('Unit | Component | InterestModal.vue', () => {
         // Then
         expect(missionName).to.equal('SCLOU - Cloud computing : enjeux, architecture et');
       });
+
+
+      it.only('should get customerName', () => {
```

Comment:
oups un it.only

Reply:
Utilisons un describe spécifique pour customerName (et pas celui de missionName)

Reply:
C'est fait.